cmd/openshift-install/create.go

```diff
@@ -125,7 +129,7 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 				return err
 			}
 
-			if a.name == "Install Config" {
+			if a.Name() == "Cluster" {
```

Comment:
you can add a postRun to cluster command ?

Reply:
> you can add a postRun to cluster command ?

My reading was that `PostRun` fires regardless of `RunE` success, and I didn't see a way to check that success from `PostRun`.  But maybe I read it wrong?

Reply:
> I'll reroll to use `PostRunE`.

Done with c898075 -> ca9ab23.